integration-libs/cdc/root/service/cdc-js.service.spec.ts

```diff
@@ -343,7 +343,7 @@ describe('CdcJsService', () => {
   });
 
   describe('onInitRegistrationHandler', () => {
-    it('should register the user', (done) => {
+    it('should register the user', () => {
```

Comment:
I suggest keeping the `done`, but calling it from the `complete` channel.
Basically, something like this:

```ts
service['onInitRegistrationHandler'](
        {
          uid: 'uid',
          password: 'password',
          firstName: 'fname',
          lastName: 'lname',
        },
        { regToken: 'TOKEN' }
      ).subscribe({
        complete: () => {
          expect(
            winRef.nativeWindow['gigya'].accounts.register
          ).toHaveBeenCalledWith({
            email: 'uid',
            password: 'password',
            profile: {
              firstName: 'fname',
              lastName: 'lname',
            },
            regToken: 'TOKEN',
            finalizeRegistration: true,
            callback: jasmine.any(Function),
          });
          done();
        },
      });
```